hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalReplicatedMapStatsImpl.java

```diff
@@ -299,6 +299,16 @@ public long getHeapCost() {
     public void setHeapCost(long heapCost) {
     }
 
+    @Probe
```

Comment:
Should we expose it as a probe if there is no useful data? Probably we may just throw `UnsupportedOperationException` here.

Reply:
No, this should have been `@Override` and not `@Probe`. Nice catch.